notebooks/develop.ipynb

```diff
@@ -530,8 +530,27 @@
       "{\n",
       "  \"amount\": \"lt\",\n",
```

Comment:
in this example the qualitative `lt` is less than what?

Reply:
See above.